package.json

```diff
@@ -36,6 +36,7 @@
     "express": "^4.17.1",
     "husky": "^3.0.3",
     "node-fetch": "^2.6.0",
+    "nodemon": "^1.19.2",
```

Comment:
remove.

Reply:
was throwing error unless I added.

Reply:
this should be a dev dependency. Could you remove, then run `yarn add nodemon -D`. In production, we will not be using nodemon

Reply:
since you fixed this in a different PR, could you remove this? https://github.com/garageScript/myProxy/pull/123

Reply:
ok